src/game/collide.c

```diff
@@ -40,10 +43,27 @@ void Collide_GetCollisionInfo(
     coll->mid_ceiling = ceiling;
     coll->mid_type = g_HeightType;
     coll->trigger = g_TriggerIndex;
-
-    int16_t tilt = Room_GetTiltType(floor, x, g_LaraItem->pos.y, z);
-    coll->tilt_z = tilt >> 8;
-    coll->tilt_x = (int8_t)tilt;
+    bool skip_slope_check = false;
+
+    // TODO Do we need to check all walkable heights or is knowing Lara is above
+    // one bridge enough? Can a slope ever be below one bridge but above
+    // another?
+    VECTOR *walkable_heights = Room_GetWalkableHeights(floor, x, ypos, z);
+    for (int32_t i = 0; i < walkable_heights->count; i++) {
+        int32_t item_height = (int32_t)walkable_heights->items[i];
+        // TODO Need check within WALL_L range? Can walkables ever be below a
+        // slope?
+        if (ABS(item_height - ypos) < WALL_L
+            && ABS(item_height - ypos) <= STEPUP_HEIGHT) {
+            skip_slope_check = true;
+        }
+    }
+    if (!skip_slope_check) {
+        int16_t tilt = Room_GetTiltType(floor, x, g_LaraItem->pos.y, z);
+        coll->tilt_z = tilt >> 8;
+        coll->tilt_x = (int8_t)tilt;
+    }
+    Vector_Clear(walkable_heights);
```

Comment:
This should've been `Vector_Free`, since we'll later override the content of this variable with a separately allocated vector.